frontend/src/navbar.tsx

```diff
@@ -1,27 +1,25 @@
-import { usePulls } from './pulls-context';
+import { usePulls, useSetFilter } from './pulls-context';
 import { Pull } from './pull';
-import Navbar from 'react-bootstrap/Navbar';
-import styled from 'styled-components';
+import { Center, Flex, Box, BoxProps, Input } from "@chakra-ui/react"
 
-const Brand = styled(Navbar.Brand)`
-   width: 200px;
-   position: absolute;
-   margin: auto;
-   left: 0;
-   right: 0;
-`;
-
-const PulldasherNav = styled(Navbar)`
-   background: var(--header-background);
-`;
-
-export default function navbar() {
+export function Navbar(props: BoxProps) {
    const pulls: Pull[] = usePulls();
+   const setPullFilter = useSetFilter();
+   const updateFilter = function(event: React.ChangeEvent<HTMLInputElement>) {
+      const pattern = event.target.value;
+      setPullFilter((pull: Pull) => {
+         return pull.title.includes(pattern);
+      });
+   };
+
    return (
-      <PulldasherNav expand="sm">
-         <Navbar.Text>{pulls.length} open</Navbar.Text>
-         <Brand>Pulldasher</Brand>
-         <Navbar.Toggle/>
-      </PulldasherNav>
+      <Center p={2} bgColor="var(--header-background)" {...props}>
+         <Flex w={1024} justify="space-between">
+            <Box alignSelf="center" w={150}>{pulls.length} open</Box>
+            <Box alignSelf="center">PULLDASHER</Box>
```

Comment:
Why tho? Why is putting it in code in a different way than we want it shown useful?

Reply:
Because it means the characters are actually correctly-cased, since the caps-ness of them is style, not official capitalization. In other words, if we referred to it as PULLDASHER in normal text, I'd be on-board with this, but we call it Pulldasher, so using CSS allows us to keep the actual text matching the official name. (FWIW, it's currently done that way.) Also, small-caps are cute:
![image](https://user-images.githubusercontent.com/1283490/157768475-ce6206f0-ecfd-45a9-af09-e542b77e5025.png)